tools/dds/realdds-py/python.cpp

```diff
@@ -218,15 +221,26 @@ PYBIND11_MODULE(NAME, m) {
                         eprosima::fastdds::dds::TypeSupport const &,
                         char const * >() )
         .def( "get_participant", &dds_topic::get_participant )
+        .def( "name", []( dds_topic const & self ) { return self->get_name(); } )
         .def( "__repr__", []( dds_topic const & self ) {
             std::ostringstream os;
             os << "<" SNAME ".topic \"" << self->get_name() << "\"";
             os << ">";
             return os.str();
         } );
 
+    using durability = eprosima::fastdds::dds::DurabilityQosPolicyKind;
+    py::enum_< durability >( m, "durability" )
+        .value( "volatile", eprosima::fastdds::dds::DurabilityQosPolicyKind::VOLATILE_DURABILITY_QOS )
+        .value( "transient_local", eprosima::fastdds::dds::DurabilityQosPolicyKind::TRANSIENT_LOCAL_DURABILITY_QOS )
+        .value( "transient", eprosima::fastdds::dds::DurabilityQosPolicyKind::TRANSIENT_DURABILITY_QOS );
```

Comment:
We are not using `TRANSIENT`, if adding values we don't use there is also `PERSISTENT` durability

Reply:
It's not about whether it's being used or not; it's about whether it's available. `PERSISTENT` is not supported, so I did not add.